### ISSUE TITLE
UniformTypeIdentifiers not loaded (MacOS)

### ISSUE THREAD

**Issue by sihellem** [keybase/client#27008]
The GUI (version Intel (x86) installed from the .io) under macOS Catalina (10.15.7) updated, and now won't launch anymore on my MacBook Pro (I had no issues to use it last week).

It blocks at:
"There was an error trying to run the install (null)."

When running the ```keybase``` command, it seems a lib is not loaded:
```
➜  ~ keybase                                                         
dyld: Library not loaded: /System/Library/Frameworks/UniformTypeIdentifiers.framework/Versions/A/UniformTypeIdentifiers
  Referenced from: /usr/local/bin/keybase
  Reason: image not found
[1]    1782 abort      keybase
```
Running ```keybase log send``` produces the same output.

Looking further, ```/System/Library/Frameworks/UniformTypeIdentifiers.framework/``` is actually not existing on my system, so I am puzzled on why Keybase would need to now load it...

**Comment by seelts** [comment]
same

**Comment by sihellem** [comment]
Solved in #2802 

https://github.com/keybase/client/issues/2802#issuecomment-1036617417

**Comment by antwal** [comment]
same problem, but i not have keybase

```
> orders-invoices@1.0.0 start
> electron-forge start

✔ Checking your system
✔ Locating application
✔ Loading configuration
✔ Preparing native dependencies [0.2s]
✔ Running generateAssets hook

dyld: Library not loaded: /System/Library/Frameworks/UniformTypeIdentifiers.framework/Versions/A/UniformTypeIdentifiers
  Referenced from: /Users/antwal/Projects/orders-invoices/orders-invoices/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
  Reason: image not found
```
